articles/rich-text-editor-reference.mdx

```diff
@@ -0,0 +1,489 @@
+  <comp.DataReferenceRow label="center" type_markdown="String">
+    This key enables a tool to center align selected text by toggling a class name for that text block. The value is the class name to align the text. The styles for this class must be defined in the `styles` file.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="right" type_markdown="String">
+    This key enables a tool to right align selected text by toggling a class name for that text block. The value is the class name to align the text. The styles for this class must be defined in the `styles` file.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+  <comp.DataReferenceRow label="justify" type_markdown="String">
+    This key enables a tool to justify selected text by toggling a class name for that text block. The value is the class name to align the text. The styles for this class must be defined in the `styles` file.
+
+    Available for the Content Editor (`_editables.content`), the Block editable region (`_editables.block`), and Rich Text inputs (`_inputs.key_name.options`).
+  </comp.DataReferenceRow>
+</comp.DataReference>
+
+The `styles` CCS file must be in plain CSS.
+
+<comp.CodeBlock language="css" source="css/editor.css">
+```
+/* Can be applied to blocks of content */
+p.callout {
+  margin: 10px;
+  border: 1px solid #f5f5f5;
+  background-color: #eee;
+}
+
+/* Can be applied to inline content */
+span.big-blue-text {
+  font-size: 2rem;
+  color: blue;
+}
+
+/* Applied to content, excluded from style dropdown */
+h2 {
+  font-family: cursive;
+}
+
+/* Applied to content, excluded from style dropdown */
+.align-left { text-align: left; }
+.align-center { text-align: center; }
+.align-right { text-align: right; }
+.align-justify { text-align: justify; }
+```
+</comp.CodeBlock>
+
+If both source and output `styles` CSS files exist, CloudCannon will default to the source file.
+
+Selectors must specify an element and one class in order to be included in the styles drop down. Styles with incompatible selectors are applied to the editor, but not shown as options.
+
+<comp.Notice info_type="info">
+  Custom styles in the Visual Editor require the same styles on your live site. If you don’t have the same styles on your live site, the class will be present but the preview will not show any styling.
+</comp.Notice>
+
+### List options
+
+The following options are available in the WYSIWYG toolbar for managing lists:
+
+<comp.MultiCodeBlock language="YAML" translate_into={["json", "JavaScript"]} source="cloudcannon.config.yml">
+``````````
+
+_editables:
+  content:
+    bulletedlist: true
+    indent: true
+    numberedlist: true
+    outdent: true
```

Comment:
We should order this one 

- bulletedlist
- numberedlist
- indent
- outdent

just to group them better